### PR TITLE
feat : wishlist screen added

### PR DESCRIPTION
**Pull request by AtulHande03a**
wish list screen added
Screenshot (84)

### CODE REVIEW

**Review by anandkumarcoder — APPROVED** [review]
Looks good. merge it

File: pages/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+            alt="sofa-ikea"
+          />
+          <span class="text-badge">Best Seller</span>
+          <i class="fa-regular fa-circle-xmark dismiss-btn"></i>
+          <i class="fa-regular fa-heart icon-badge"></i>
+          <div class="card-rating flex flex-center">
+            <span class="card-rating-star">4.5</span>
+            <i class="fa-solid fa-star rating-star-icon"></i>
+          </div>
+          <div class="card-addtocart orange-btn hide">
```
Comment (xii-vi) [inline pages/wishlist.html]: Follow proper class name i.e here addtocart should have hypen between them.

**Review by xii-vi — APPROVED** [review]
Left a comment.
Check it out before merging.

File: pages/wishlist.html (hunk 1 above)
Comment (AtulHande03a) [reply pages/wishlist.html]: yes I will make changes to this. Thanks for pointing this issue

**Pull request merged**